css/style.css

```diff
@@ -0,0 +1,152 @@
+* {
+	box-sizing: border-box;
+	font-family: "Times New Roman", Times, serif;
+}
+
+h1 {
+	margin-bottom: 40px;
+	text-align: center;
+	font-weight: bold;
+	font-family: 'Alice', serif;
+}
+
+section {
+	position: relative;
+	font-weight: bold;
+	}
+
+#boxchicken {
+	background-color: #778899;
+	font-family: 'Alice', serif;
+	position: absolute;
+	border-style: solid;
+	border-width: 1px;
+	font-size: 120%;
+	padding: 5px 30px 5px 30px;
+	top: 0;
+	right: 0;
+	}	
+
+#boxbeef {
+	background-color: #FFB6C1;
+	font-family: 'Alice', serif;
+	font-size: 120%;
+	border-color: 	#000000;
+	border-style: solid;
+	border-width: 1px;
+	padding: 5px 30px 5px 30px;
+	position: absolute;
+	top: 0;
+	right: 0;
+	}	
+
+#boxsushi {
+	background-color: #20B2AA;
+	font-family: 'Alice', serif;
+	font-size: 120%;
+	border-color: 	#000000;
+	border-style: solid;
+	border-width: 1px;
+	padding: 5px 30px 5px 30px;
+	position: absolute;
+	top: 0;
+	right: 0;
+	}
+
+#sectionchicken {
+	background-color: #ADD8E6;
+	font-family: 'Alice', serif;
+	border-color: 	#000000;
+	border-style: solid;
+	border-width: 1px;
+	padding: 50px 15px 15px 15px;	
+	margin: 10px 5px 10px 10px;
+	float: left;
+	width: 32%;
```

Comment:
Your calc function below subtracts 20px from 33%, yet here you specify the width as 32%.

Reply:
I have made the correction, thank you! You're da bomb